src/pom.xml

```diff
@@ -738,6 +738,16 @@
     <artifactId>commons-httpclient</artifactId>
     <version>3.1</version>
    </dependency>
+   <dependency>
+    <groupId>org.apache.httpcomponents</groupId>
+    <artifactId>httpclient</artifactId>
+    <version>4.5.5</version>
```

Comment:
This was already declared below in the same file.
Mind, the httpcomponents are used by some community modules (authkey, geogig, onelogin) this could cause packaging and installation issues (duplicate jars with potentially different version). Maybe it's best to remove the version from the community modules now that v4 is used by core, so that they are all aligned.